presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -328,120 +324,254 @@ Statistical Aggregate Functions
     Returns the excess kurtosis of all input values. Unbiased estimate using
     the following expression:
 
-    .. code-block:: none
+    .. math::
 
-        kurtosis(x) = n(n+1)/((n-1)(n-2)(n-3))sum[(x_i-mean)^4]/stddev(x)^4-3(n-1)^2/((n-2)(n-3))
+        \mathrm{kurtosis}(x) = {n(n+1) \over (n-1)(n-2)(n-3)} { \sum[(x_i-\mu)^4] \over \sigma^4} -3{ (n-1)^2 \over (n-2)(n-3) },
 
-.. function:: classification_miss_rate(buckets, y, x, weight) -> array<double>
+   where :math:`\mu` is the mean, and :math:`\sigma` is the standard deviation.
 
-    Computes the miss-rate part of the receiver operator curve with up to ``buckets`` number of buckets. Returns
-    an array of miss-rate values. ``y`` should be a boolean outcome value; ``x`` should be predictions, each
-    between 0 and 1; ``weight`` should be non-negative values, indicating the weight of the instance.
+.. function:: regr_intercept(y, x) -> double
 
-.. function:: classification_fall_out(buckets, y, x, weight) -> array<double>
+    Returns linear regression intercept of input values. ``y`` is the dependent
+    value. ``x`` is the independent value.
+
+.. function:: regr_slope(y, x) -> double
+
+    Returns linear regression slope of input values. ``y`` is the dependent
+    value. ``x`` is the independent value.
+
+.. function:: skewness(x) -> double
+
+    Returns the skewness of all input values.
+
+.. function:: stddev(x) -> double
+
+    This is an alias for :func:`stddev_samp`.
+
+.. function:: stddev_pop(x) -> double
+
+    Returns the population standard deviation of all input values.
+
+.. function:: stddev_samp(x) -> double
+
+    Returns the sample standard deviation of all input values.
+
+.. function:: variance(x) -> double
+
+    This is an alias for :func:`var_samp`.
+
+.. function:: var_pop(x) -> double
+
+    Returns the population variance of all input values.
+
+.. function:: var_samp(x) -> double
+
+    Returns the sample variance of all input values.
 
-    Computes the fall-out part of the receiver operator curve with up to ``buckets`` number of buckets. Returns
-    an array of miss-rate values. ``y`` should be a boolean outcome value; ``x`` should be predictions, each
-    between 0 and 1; ``weight`` should be non-negative values, indicating the weight of the instance.
 
-    To get an ROC map, use this in conjunction with :func:`classification_recall`:
+Classification Metrics Aggregate Functions
+------------------------------------------
+
+The following functions each measure how some metric of a binary
+`confusion matrix <https://en.wikipedia.org/wiki/Confusion_matrix>`_ changes as a function of a function of
```

Comment:
It's not clear to me whether there's an extra "a function of". Both makes sense. There's probably a less confusing way to describe this?

Reply:
Thanks, @rongrong - moving this to a different diff.